.eslintrc.js

```diff
@@ -947,7 +947,64 @@ module.exports = {
       },
     },
     {
-      // typescript and javascript for front and back end
+      // typescript for /public and /common
+      files: ['x-pack/plugins/lists/public/*.{ts,tsx}', 'x-pack/plugins/lists/common/*.{ts,tsx}'],
+      rules: {
```

Comment:
Just a note; if we think these rules are beneficial - it would be good to see if we could get a consensus to adopt them universally.

Reply:
Our team's made efforts to follow some functional patterns and some of these plugin specific overrides are related to that. Would be awesome to hear others opinions.